src/tribler-gui/tribler_gui/core_manager.py

```diff
@@ -101,9 +112,10 @@ def start_tribler_core(self, core_args=None, core_env=None):
 
         self.core_process = QProcess()
         self.core_process.setProcessEnvironment(core_env)
-        self.core_process.setReadChannel(QProcess.StandardOutput)
-        self.core_process.setProcessChannelMode(QProcess.MergedChannels)
-        connect(self.core_process.readyRead, self.on_core_read_ready)
+        self.core_process.setReadChannel(QProcess.StandardOutput)  # not necessary?
```

Comment:
**Nit:**  this comment is not useful for future readers and may be deleted

Reply:
I suspect this line of code can be removed, but it requires further investigation.
It may be useful for readers to know of the code that the presence of this line does not have obvious reason, in case they want to refactor the code.
In principle, I can remove the line right now in hope that it will not break something non-obvious.